app/templates/components/widgets/forms/ticket-input.hbs

```diff
@@ -11,11 +11,11 @@
       </div>
     {{else if (eq ticket.type 'donation')}}
       <div class="two fields">
-        <div class="field">
-          {{input type='number' name='min_price' placeholder=(t 'Min') value=ticket.minOrder}}
+        <div class="field {{unless (gte ticket.maxPrice ticket.minPrice) 'error'}}" >
```

Comment:
@niranjan94 Travis fails by saying   `14:35  error  Using {{unless}} in combination with other helpers should be avoided. MaxHelpers: 0  simple-unless`.

Reply:
Okay. Just got it. ES6 style classes.